assets/js/phoenix_live_view.js

```diff
@@ -511,11 +511,28 @@ export class LiveSocket {
     for(let type of ["change", "input"]){
       this.on(type, e => {
         let input = e.target
-        let key = input.type === "checkbox" ? "checked" : "value"
-        if(this.prevInput === input && this.prevValue === input[key]){ return }
+
+        let value
+        if (input.type === "checkbox") {
+          value = input["checked"]
+        }
+        else if (input.type === "select-multiple") {
+          let selectedOptions = input["selectedOptions"]
+          let values = []
+          for (let i = 0; i < selectedOptions.length; i++) {
+            values.push(selectedOptions.item(i).value)
+          }
+          // Convert to a string for easier array comparison
+          value = JSON.stringify(values)
```

Comment:
Instead of `JSON.stringify`, we could do some thing like this:

```js
var selectedOptions = input["selectedOptions"]
for (let i = 0; i < selectedOptions.length; i++) {
  value += selectedOptions.item(i).value
}

```

This would avoid creating an extra data structure in the system and better perf.  What do you think?

https://jsperf.com/array-stringify/1

Reply:
Great call out on performance of simply concatenating. I’ll update the it with your suggestion.

Reply:
@snewcomer Ready for another look. Slightly deviates from the suggestion keep an array-looking string.